db/migrate/20171209171303_rename_events_to_event_instances.rb

```diff
@@ -0,0 +1,66 @@
+class RenameEventsToEventInstances < ActiveRecord::Migration[5.1]
+  # Stub models for migration in case they get renamed in the future
```

Comment:
I don't understand why you'd need to do this — isn't it that migrations are always run in order in a repo so at this point in time the table names will always be what they are now?

Or do you mean if the table names were to be changed within this commit/PR?

Reply:
Right, got it! Of course. We're stubbing models, not the table names! Thanks.